src/components/resume/educationResumeEntry.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+  export let i: number;
+  export let name: string;
+  export let type: string;
+  export let degree: string;
+  export let startDate: string;
+  export let endDate: string;
+</script>
+
+<p id={`type_degree_name_${i}`}>{type}, {degree}, {name}</p>
+<p id={`educaiton_name_${i}`} class={'education-date'}>{startDate} - {endDate}</p>
```

Comment:
Add spans in this text to assign ids to each of the properties for ease of testing